package.json

```diff
@@ -0,0 +1,108 @@
+{
+  "name": "redux-debounce-field",
```

Comment:
Wouldn't this be better named `redux-form-debounce-field`, as that is what it is actually a plugin of.

Reply:
renamed, we also rename the repo